emails/preview-v3.html

```
@@ -0,0 +1,92 @@
+  .preview-nav h2 span { background: linear-gradient(135deg, #06b6d4 0%, #8B5CF6 50%, #9333ea 100%); -webkit-background-clip: text; -webkit-text-fill-color: transparent; }
+  .frame-wrap { max-width: 660px; margin: 32px auto; }
+  iframe { width: 100%; height: 900px; border: 1px solid rgba(100,116,139,0.12); border-radius: 16px; background: #0a0a0f; }
+  .device-toggle { display: flex; gap: 8px; margin-left: auto; }
+  .device-toggle button { background: rgba(34,211,238,0.08); color: #22d3ee; border: 1px solid rgba(34,211,238,0.15); font-size: 12px; }
+</style>
+</head>
+<body>
+
+<div class="preview-nav">
+  <h2><span>FrankX.AI</span> Email Design System v3.0</h2>
+  <button class="active" onclick="show('welcome')">Welcome</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)